### PR TITLE
Lodash assign for react native support

### PR DESCRIPTION
**Pull request by rjerue**
Love this library for use of Polyglot with web development. I'm working on a react native project with shared items to a react.js project and wanted to use this library for i18n. However, Object.assign in React-Native does not allow for enumerable keys to be prototypes on objects, and they are for Polygots.

The solution is to change the {... p, ...} (which is Object.assign) to use Lodash's assign instead.

### CODE REVIEW

File: src/selectors.js [hunk 1]
```diff
@@ -94,14 +95,7 @@ const createGetP = (polyglotOptions) => {
...
             }
-            return {
-                ...p,
-                t,
-                tc,
-                tt,
-                tu,
-                tm,
-            };
+            return assign(p, t, tc, tt, tu, tm);
```
Comment (guillaumearm) [inline src/selectors.js]: from https://lodash.com/docs/4.17.5#assign : 
> This method mutates object

You need to `assign({}, ...)`.
Anyway, `t`, `tc`, `tt`, `tu` and `tm` are functions.

The correct implementation is next : 
```js
return assign({}, p, t, tc, tt, tu, tm);
```

**Review by guillaumearm — CHANGES_REQUESTED** [review]
Hi, thanks for contributing to `redux-polyglot`.

Since our CI is broken, we need to run tests before push.
At this time, tests are broken, please fix them.

In addition, please install `npm@5` and run a `npm install` in order to update `package-lock.json`

**Comment by rjerue** [comment]
I'll look into fixing the jest tests... ~As of now, Jest isn't detecting that anything is found.~ Running from wrong folder...

**Comment by rjerue** [comment]
Changes made, passes all unit tests now. Still works in my app too.

Hope other React-Native users find use for this.

**Review by guillaumearm — APPROVED** [review]

**Comment by guillaumearm** [comment]
Nice work @rjerue, You are now part of [redux-polyglot contributors](https://github.com/Tiqa/redux-polyglot/graphs/contributors).

Anyway, I just released `redux-polyglot@0.6.5` on `latest` npm dist-tag.

**Pull request merged**